### PR TITLE
Start zombiejs server before basic auth information is set

### PR DESCRIPTION
**Pull request by berliner**
Related to https://github.com/minkphp/MinkZombieDriver/issues/179

### CODE REVIEW

File: src/ZombieDriver.php [hunk 1]
```diff
@@ -167,6 +167,11 @@ public function back()
      */
     public function setBasicAuth($user, $password)
     {
+        // Zombie expects session to be started prior to setting auth info.
+        if (!$this->isStarted()) {
+          $this->start();
```
Comment (stof) [inline src/ZombieDriver.php]: please use 4 spaces for the indentation to be consistent with the project

Comment (berliner) [inline src/ZombieDriver.php]: done

**Comment by berliner** [comment]
Hm, the failings seem unrelated to my changes.

**Comment by berliner** [comment]
@stof Is there anything more I can do to move this forward?

**Pull request merged**